insights/parsers/tests/test_hammer_task_list.py

```diff
@@ -0,0 +1,20 @@
+from ...parsers import hammer_task_list
+from ...tests import context_wrap
+
+import doctest
+
+
+def test_HTL_doc_examples():
+    failed, tested = doctest.testmod(hammer_task_list)
+    assert failed == 0
```

Comment:
@PaulWay I'm assuming that you are using your Examples as your positive tests?

Reply:
Yep, definitely!  No need to write two tests for the one thing.